site/en/docs/web-platform/navigation-api/index.md

```diff
@@ -378,23 +357,23 @@ E.g., as a developer, I could:
 - allow the user to complete their work (or go Back)
 - remove a history entry on completion of a task
 
-This could be perfect for temporary modals or interstitals: the new URL is something that a user can use the Back gesture to leave from, but they then cannot accidentaly go Forward to open it again (because the entry has been removed).
+This could be perfect for temporary modals or interstitials: the new URL is something that a user can use the Back gesture to leave from, but they then cannot accidentally go Forward to open it again (because the entry has been removed).
 This is just not possible with the current History API.
 
-## Try the App History API
+## Try the Navigation API
 
-You can try the App History API in Chrome 95 and above by enabling the "Experimental Web Platform features" flag.
+You can try the Navigation API in Chrome 95 and above by enabling the "Experimental Web Platform features" flag.
 You can also [try out a demo][demo] by [Domenic Denicola][domenic].
 
 We're especially eager for feedback on issues labelled with ["feedback wanted"][feedback-wanted] on GitHub.
```

Comment:
[This list](https://github.com/WICG/navigation-api/issues?q=is%3Aissue+is%3Aopen+label%3A%22feedback+wanted%22) is pretty sparse these days, so maybe delete it.